servicebox-app/src/test/java/io/halkyon/ClaimingJobServiceTest.java

```diff
@@ -51,7 +52,7 @@ public void testJobShouldMarkClaimAsErrorAfterMaxAttemptsExceeded(){
                 .then()
                 .statusCode(200)
                 .body("status", is(ClaimStatus.CLAIMED.toString()))
```

Comment:
According to the PR opened to use less status then CLAIMED should become BIND ?

Reply:
That's part of the a different pull request.